Dockerfile

```diff
@@ -1,3 +1,5 @@
 FROM yastdevel/ruby
 COPY . /usr/src/app
 
+RUN zypper -n in yast2-ntp-client
```

Comment:
As it is only needed for running the tests it's a better idea to mock whole modules, see this example https://github.com/yast/yast-installation/blob/master/test/test_helper.rb#L19-L28.

This means the package would be needed also during RPM build and that would make the build path at OBS longer.

Reply:
@lslezak thank you!

I'll to revert it and do something similar to the linked example.

Reply:
@lslezak I am not sure that, in this case, it affects OBS. `yast2-add-on` depends on `autoyast2-installation` which depends on `yast2-ntp-client` yet.